src/main/frontend/components/publishing.cljs

```diff
@@ -0,0 +1,136 @@
+(ns frontend.components.publishing
+  (:require [rum.core :as rum]
+            [frontend.context.i18n :as i18n]
+            [frontend.db :as db]
+            [frontend.state :as state]
+            [frontend.util :as util]
+            [reitit.frontend.easy :as rfe]
+            [frontend.db-mixins :as db-mixins]
+            [frontend.config :as config]
+            [lambdaisland.glogi :as log]
+            [promesa.core :as p]
+            [frontend.handler.page :as page-handler]
+            [frontend.handler.notification :as notification]
+            [frontend.ui :as ui]
+            [frontend.components.svg :as svg]
+            [frontend.handler.project :as project-handler]))
+
+
+(defn project
+  [editor-state current-project pages]
+  (rum/with-context [[t] i18n/*tongue-context*]
+    (if (= :display @editor-state)
+      (when current-project
+        [:div.cp__publishing-pj
+         [:span.cp__publishing-pj-name current-project]
+         [:span.cp__publishing-edit
+          {:on-click
+           (fn [_]
+             (reset! editor-state :editor))}
+          (t :publishing/edit)]])
+      [:div.flex.cp__publishing_pj_edit
+       [:input#cp__publishing-project-input
+        {:placeholder current-project
```

Comment:
We can add a `:autoFocus true` attribute so that users don't need to click the input themselve.